smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/extension/EmitterImpl.java

```diff
@@ -123,33 +124,34 @@ boolean isConnected() {
         if (msg == null) {
             throw new IllegalArgumentException("`null` is not a valid value");
         }
-        FlowableEmitter<Message<? extends T>> emitter = verify(internal);
+        FlowableEmitter<Message<? extends T>> emitter = verify(internal, name);
         emitter.onNext(msg);
 
     }
 
-    static <T> FlowableEmitter<Message<? extends T>> verify(AtomicReference<FlowableEmitter<Message<? extends T>>> reference) {
+    static <T> FlowableEmitter<Message<? extends T>> verify(AtomicReference<FlowableEmitter<Message<? extends T>>> reference,
+            String name) {
         FlowableEmitter<Message<? extends T>> emitter = reference.get();
         if (emitter == null) {
-            throw new IllegalStateException("Stream not yet connected");
+            throw new IllegalStateException("No one subscribed to channel " + name);
```

Comment:
"No one" or "Nothing"?

Reply:
How about "No subscriber for the channel "?

Reply:
Much better, let me change this!